python_modules/airline-demo/airline_demo/pipelines.py

```diff
@@ -66,21 +77,71 @@ def _db_load(data_frame, table_name, resources):
             'No implementation for db_dialect "{db_dialect}"'.format(db_dialect=db_dialect)
         )
 
-def define_lambda_resource(func):
-    return ResourceDefinition(lambda _info: func())
+
+def define_lambda_resource(func, *args, **kwargs):
+    return ResourceDefinition(lambda _info: func(*args, **kwargs))
+
 
 def define_value_resource(value):
     return ResourceDefinition(lambda _info: value)
 
+
 def define_null_resource():
     return define_value_resource(None)
 
+
 def define_string_resource():
     return ResourceDefinition(
         resource_fn=lambda info: info.config,
         config_field=types.Field(types.String),
     )
 
+
+class TempfileManager(object):
```

Comment:
We're definitely going to want to add an option to not cleanup files for unittesting purposes. But we can add that when we get to it.

Reply:
👍